test/benchmark/fuse_db/bin/run_mysql.sh

```
@@ -0,0 +1,116 @@
+  kill -0 $LPID 2>/dev/null || { say "loader exited early"; break; }
+  sleep 0.2
+done
+LB=$(cat "$PROG" 2>/dev/null || echo 0)
+say "durable committed lower-bound at crash = $LB rows; crashing now"
+hard_kill_all
+kill -9 $LPID 2>/dev/null; wait $LPID 2>/dev/null
+sleep 1
+cluster_start || exit 1; mount_start || exit 1
+say "restarting mysqld -> InnoDB recovery; in-flight txn must roll back, committed must survive"
+mysql_start || { echo "recovery failed"; tail -30 "$LOGS/mysqld.log"; chk 1 "C/crash-during-write"; }
+if mq -e "SELECT 1" >/dev/null 2>&1; then mysql_verify $LB atleast; chk $? "C/crash-during-write"; fi
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)